src/SPC/builder/unix/UnixBuilderBase.php

```diff
@@ -210,21 +211,71 @@ protected function sanityCheck(int $build_target): void
     /**
      * Deploy the binary file to the build bin path.
      *
-     * @param int $type Type integer, one of BUILD_TARGET_CLI, BUILD_TARGET_MICRO, BUILD_TARGET_FPM
+     * @param int $type Type integer, one of BUILD_TARGET_CLI, BUILD_TARGET_MICRO, BUILD_TARGET_FPM, BUILD_TARGET_CGI, BUILD_TARGET_FRANKENPHP
      */
-    protected function deployBinary(int $type): bool
+    protected function deployBinary(int $type): void
     {
+        FileSystem::createDir(BUILD_BIN_PATH);
+        $copy_files = [];
         $src = match ($type) {
             BUILD_TARGET_CLI => SOURCE_PATH . '/php-src/sapi/cli/php',
             BUILD_TARGET_MICRO => SOURCE_PATH . '/php-src/sapi/micro/micro.sfx',
             BUILD_TARGET_FPM => SOURCE_PATH . '/php-src/sapi/fpm/php-fpm',
             BUILD_TARGET_CGI => SOURCE_PATH . '/php-src/sapi/cgi/php-cgi',
+            BUILD_TARGET_FRANKENPHP => BUILD_BIN_PATH . '/frankenphp',
```

Comment:
we also need target embed for libphp(-postfix).so

Reply:
I'm not talking about the `-release` of LDFLAGS, but the patch for soname. It not quite belongs to `build` though. 

I understand the concerns about separating it out, as this is libtool's approach, but it's better suited to be implemented as a standalone patches.

As for the current version for my perspective, I wouldn't exclude it from the list of default internal patches, but it certainly not appropriate to be here.